### ISSUE TITLE
Install command help has misleading text for --role

### ISSUE THREAD

**Issue by jakolehm** [k0sproject/k0s#593]
**Version**
```
$ k0s version
v0.9.1
```
**What happened?**
`k0s install --help` says
> --role string   node role (possible values: server or worker. In a single-node setup, a worker role should be used) (default "server")

which is misleading (or wrong) because docs say:

> enable-worker (single-node) setup is not supported.

**Comment by jnummelin** [comment]
IMO we need to come up a way to make the install command to take in any supported flags and args for the server/worker commands. This way we do not need to copy all flags to install cmd.

By making the install command take in anything supported we can make it easily "consumable" for the `get.k0s.sh` script too. So something like:
```
curl ... | K0S_INSTALL_EXEC="worker --token-file /my/token"
```
would run the installer script and part of that run the `k0s install worker --token-file /my/token` to setup the systemd/openrc unit

**Comment by trawler** [comment]
Hey, jumping in a bit late, but if I recall correctly, we were discussing the whole purpose of the “install” command with @ncopa way back, and wanted to have the install command support users for setting up the service in the officially recommended way. I.e, not run all services as root, etc. 
Given this assumption, I supposed that the single node setup isn’t in fact a “recommended” method. By supporting it in the install command we are actually making it easier to run it that way. Is that a route we want to take?
I agree that the help output should be clarified, but I am not yet convinced we should modify the whole behavior of the install command.

**Comment by jasmingacic** [comment]
By merging #594 I believe this shouldn't be an issue anymore.
@jakolehm can you confirm this so we can close this issue?

**Comment by jasmingacic** [comment]
Fixed #594
